data/src/test/java/org/apache/iceberg/data/DataTest.java

```diff
@@ -56,7 +56,8 @@
       required(114, "dec_9_0", Types.DecimalType.of(9, 0)),
       required(115, "dec_11_2", Types.DecimalType.of(11, 2)),
       required(116, "dec_38_10", Types.DecimalType.of(38, 10)), // maximum precision
-      required(117, "time", Types.TimeType.get())
+      required(117, "time", Types.TimeType.get()),
+      required(118, "time_ms",  Types.TimeType.get())
```

Comment:
Iceberg won't actually write data in milliseconds, so we would need to write a Parquet file using the annotation and confirm that a `time` column is correct when reading.

Can you remove this change?

Reply:
done.

Reply:
@rdblue do we have any boilerplate code I can use to do that?